libsrc/osic1p/clrscr.s

```diff
@@ -0,0 +1,31 @@
+;
+; void clrscr (void);
+;
+        .export         _clrscr
+        .import         plot
+        .include        "extzp.inc"
+        .include        "osic1p.inc"
+
+; Adapted from the Challenger Character Graphics
+; Reference Manual, "2.3.3 MACHINE LANGUAGE SCREEN CLEAR"
+; This is self-modifying code!
+BANKS = VIDEORAMSIZE / $100
+
+_clrscr:
+        lda       #$20          ; ' '

```

Comment:
I'm not going to keep this pull request further from being merged but please be aware that the assembler allows to directly define ASCII constants...